### PR TITLE
[#1090] Add fallback image to channels

### PR DESCRIPTION
**Pull request by thorstenairy**

### CODE REVIEW

File: frontend/ui/src/services/image/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import _, {SyntheticEvent} from 'react';
+
+export const fallbackImage = (event: SyntheticEvent<HTMLImageElement, Event>, channelSource: string) => {
```
Comment (lucapette) [inline frontend/ui/src/services/image/index.ts:3]: source better name

File: frontend/ui/src/services/image/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  event.currentTarget.src = `https://s3.amazonaws.com/assets.airy.co/${channelSource}_avatar.svg`;
+  event.currentTarget.alt = 'fallback image';
```
Comment (lucapette) [inline frontend/ui/src/services/image/index.ts:5]: `default ${source} image` maybe?

**Review by lucapette — COMMENTED** [review]
Minor comments :)

File: frontend/ui/src/services/image/index.ts (hunk 2 above)
Comment (thorstenairy) [reply frontend/ui/src/services/image/index.ts:5]: as we use the _avatar name exactly for those images, I would keep it.

**Review by AitorAlgorta — APPROVED** [review]

**Pull request merged**